lib/components/SectionHeading.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+import Heading from './typography/Heading';
+import SubHeading from './typography/SubHeading';
+import ModelSchematic from './ModelSchematic';
+
+const styles = theme => ({
+  container: {
+    width: '100%',
+    display: 'flex',
+    alignItems: 'flex-end',
+    paddingBottom: '1rem',
+  },
+  flex: {
+    flexGrow: 1,
+  },
+});
+
+const SectionHeading = ({ classes, heading, subheading, entities }) => {
+  return (
+    <React.Fragment>
+      <hr />
+      <div className={classes.container}>
+        <div>
+          <Heading>{heading}</Heading>
+          <SubHeading>{subheading}</SubHeading>
+        </div>
+        <div className={classes.flex} />
+        <ModelSchematic entities={entities} />
```

Comment:
We could decouple `SectionHeading` from `ModelSchematic` by using `children`. For example:
```jsx
<SectionHeading>
  <ModelSchematic/>
</SectionHeading>
```
and inside the render of `SectionHeading` we could use children:
```jsx
const SectionHeading = ({ classes, heading, subheading, entities, children }) => {
  return (
    <React.Fragment>
      <hr />
      <div className={classes.container}>
        <div>
          <Heading>{heading}</Heading>
          <SubHeading>{subheading}</SubHeading>
        </div>
        <div className={classes.flex} />
        {children}
      </div>
    </React.Fragment>
  );
};
```

This way, users are not forced to display a model schematics when they just want to use a `SectionHeader`